iso-authoryear.cbx

```diff
@@ -1,14 +1,18 @@
 \ProvidesFile{iso-authoryear.cbx}[2017/04/25 v0.3.2 biblatex citation style]
 
+% Use only one name in citation to be consistent
 \ExecuteBibliographyOptions{%
-   maxcitenames=1
-  ,mincitenames=1
+   maxcitenames=1,
+   mincitenames=1,
 }
 
 \RequireCitationStyle{authoryear}
 \RequireCitationStyle{iso-fullcite}
 
+% Delimiter between creator name and year in citation
 \renewcommand*{\nameyeardelim}{\addcomma\addspace}
```

Comment:
`nameyeardelim` is context sensitive now, so could be made into
```suggestion
\DeclareDelimFormat{nameyeardelim}{\addcomma\space}
```
Using `\DeclareDelimFormat` and friends of course has backwards compatibility implications. (`\DeclareDelimFormat` was introduced in May 2016).